### DISCUSSION TITLE
Problem about Clean Vehicles Directive (CVD)

### DISCUSSION THREAD

**Discussion by fernarr** [OP-TED/eForms-SDK#395] [Q&A] [answered]
Hello,
I have some doubts about the Clean Vehicles Directive (CVD).

In the documentation I have seen that the number of different types of vehicles (vehicles, clean vehicles or zero emission vehicles) can be indicated in different categories (M1, M2, Bus, N1,…).

For example, it can be said that the following exist:

- 3 vehicles clean in category M1
- 2 vehicles zero emissions in category M1
- 1 vehicle clean in category M2
- 4 vehicles zero emissions in category M2

In XML, in this way:

```xml
<efac:StrategicProcurementInformation>
	<efbc:ProcurementCategoryCode listName="cvd-contract-type">veh-acq</efbc:ProcurementCategoryCode>
	<efac:ProcurementDetails>
		<efbc:AssetCategoryCode listName="vehicle-category">m1</efbc:AssetCategoryCode>
		<efac:StrategicProcurementStatistics>
			<efbc:StatisticsCode listName="vehicles">vehicles-clean</efbc:StatisticsCode>
			<efbc:StatisticsNumeric>3</efbc:StatisticsNumeric>
		</efac:StrategicProcurementStatistics>
		<efac:StrategicProcurementStatistics>
			<efbc:StatisticsCode listName="vehicles">vehicles-zero-emission</efbc:StatisticsCode>
			<efbc:StatisticsNumeric>2</efbc:StatisticsNumeric>
		</efac:StrategicProcurementStatistics>
	</efac:ProcurementDetails>
	<efac:ProcurementDetails>
		<efbc:AssetCategoryCode listName="vehicle-category">m2</efbc:AssetCategoryCode>
		<efac:StrategicProcurementStatistics>
			<efbc:StatisticsCode listName="vehicles">vehicles-clean</efbc:StatisticsCode>
			<efbc:StatisticsNumeric>1</efbc:StatisticsNumeric>
		</efac:StrategicProcurementStatistics>
		<efac:StrategicProcurementStatistics>
			<efbc:StatisticsCode listName="vehicles">vehicles-zero-emission</efbc:StatisticsCode>
			<efbc:StatisticsNumeric>4</efbc:StatisticsNumeric>
		</efac:StrategicProcurementStatistics>
	</efac:ProcurementDetails>
</efac:StrategicProcurementInformation>
```

However, in eNotices2, I am only able to indicate one number of one type in a result, for example, 3 vehicles clean:

![image](https://user-images.githubusercontent.com/3096869/228672946-b3dafb27-9e6c-4875-9be0-36fe3cd1569e.png)

Are there limitations of data in eNotices2 web interface? Could I indicate de 10 vehicles of the example in some way in eNotices2 web interface? How can I indicate de category (M1, M2, Bus, …) in eNotices web interface?

Nor have I been able to locate where in eNotices2 web I can indicate the values ​​of the type of contract, field BT-735.

Thanks for the answer.

**Comment by ferrakaop** [comment, marked as answer]
The new fields from the 2022 amendment to the eForms regulation, including some additional CVD fields, have not yet been added to eNotices2. They are currently planned to be added for SDK 1.8.  The fields are available in the XML so eSenders can submit them and the TED Viewer will display them from SDK 1.7 onwards (and will also be patched to older SDK versions).
